### DISCUSSION TITLE
Unable to open with macOS 15.1.1

### DISCUSSION THREAD

**Discussion by nickshaw0609** [deskflow/deskflow#7997] [Q&A] [answered]
when I installed deskflow 1.17.2, I can't open it because of the error below, it says that my file was broken and can't be opened
![QQ20241221-174323](https://github.com/user-attachments/assets/45f74c50-6af1-46a6-b954-9333ba7aa306)

**Comment by nickshaw0609** [comment, marked as answer]
Installing on macOS

When you install Deskflow on macOS, you need to allow accessibility access (Privacy & Security) to both the Deskflow app and the deskflow process.

If using Sequoia, you may also need to allow Deskflow under Local Network‍ settings (Privacy & Security). When prompted by the OS, go to the settings and enable the access.

If you are upgrading and you already have Deskflow or deskflow on the allowed list you will need to manually remove them before accessibility access can be granted to the new version.

macOS users who download directly from releases may need to run xattr -c /Applications/Deskflow.app after copying the app to the Applications dir.

It is recommend to install Deskflow using [Homebrew](https://brew.sh/) from our [homebrew-tap](https://github.com/deskflow/homebrew-tap)

To add our tap, run:

brew tap deskflow/homebrew-tap

Then install either:

Stable: brew install deskflow
Continuous: brew install deskflow-dev

**Comment by yabo-san** [comment]
Problem still persists even after reinstalling with brew and enabling necessary permissions. 
<img width="458" alt="image" src="https://github.com/user-attachments/assets/85d1977c-58be-45e2-8dfb-303aa245c660" />

**Reply by sithlord48** [reply]
You should not have all of those only the "Deskflow" one with the icon, so thats kinda odd.  Remove them all and deskflow reboot the machine and reinstall deskflow, When you start deskflow with none of them in the list you will get a prompt  to allow it access.. When using as a client you may get an additional prompt the first time you mouse enters the mac's screen . you can use the mac's keyboard to accept that one . Hopefully that gets it working for you

**Reply by yabo-san** [reply]
it looks like it's stuck asking for permissions. any advice on how to move forward?
<img width="478" alt="image" src="https://github.com/user-attachments/assets/ac3b2279-3d6b-40b1-88b5-481d09670e60" />

**Reply by sithlord48** [reply]
Has it opened before ? is this dialog after you start the server / client ? or when you start the app ? and be sure if it was on the list you have uninstalled and rebooted and also make sure its off the list after you have restarted.

**Reply by yabo-san** [reply]
I had it working a couple months back. this popup happens when i start the app. and yes i followed your previous instructions.

**Reply by sithlord48** [reply]
macos can be a bit difficult with those settings sometimes. I had an issue like this with xcode and i had to reset all the permissions on my machine using a terminal command before i was able to give it the permission it needed to run correctly.

**Reply by yabo-san** [reply]
I ended up doing that actually
https://symless.com/synergy/help/how-to-fix-macos-security-and-privacy-accessibility-settings
